analyzer/config/json/json.go

```diff
@@ -30,21 +31,43 @@ func NewConfigAnalyzer(filePattern *regexp.Regexp) ConfigAnalyzer {
 	}
 }
 
-func (a ConfigAnalyzer) Analyze(_ context.Context, input analyzer.AnalysisInput) (*analyzer.AnalysisResult, error) {
+func (a ConfigAnalyzer) Analyze(ctx context.Context, input analyzer.AnalysisInput) (*analyzer.AnalysisResult, error) {
 	var parsed interface{}
-	if err := json.NewDecoder(input.Content).Decode(&parsed); err != nil {
+	var confType string
+	err := json.NewDecoder(input.Content).Decode(&parsed)
+	if err != nil {
 		return nil, xerrors.Errorf("unable to decode JSON (%s): %w", input.FilePath, err)
 	}
 
-	return &analyzer.AnalysisResult{
-		Configs: []types.Config{
-			{
-				Type:     types.JSON,
-				FilePath: input.FilePath,
-				Content:  parsed,
+	if configs, ok := parsed.([]interface{}); ok { // json input is array
```

Comment:
JSON doesn't need this sort of process as it is supported by YAML only.

Reply:
I've updated logic with changes from man branch. Can you take a look?